date/date_c.php

```diff
@@ -93,15 +93,15 @@ public function diff($targetObject, $absolute = false);
      * @param string $format <p>
      * Format accepted by  {@link https://secure.php.net/manual/en/function.date.php date()}.
      * </p>
-     * @return string
+     * @return string|false
      * Returns the formatted date string on success or <b>FALSE</b> on failure.
      */
     public function format($format);
 
     /**
      * (PHP 5 &gt;=5.5.0)<br/>
      * Returns the timezone offset
-     * @return int
+     * @return int|false
```

Comment:
According to https://www.php.net/manual/en/datetime.getoffset.php `false` could be returned only for PHP < 8. Please use #LanguageLevelTypeAware attribute to set return type in this case

Reply:
Done. I edited the description to explain the return type for PHP8 as well.

Reply:
Please also add `#LanguageLevelTypeAware` to imports with `use JetBrains\PhpStorm\Internal\LanguageLevelTypeAware;` as currently class is marked as undefined in the code